includes/ziti/ziti.h

```diff
@@ -1001,6 +1001,78 @@ extern void ziti_mfa_auth(ziti_context ztx, const char *code, ziti_mfa_cb auth_c
 ZITI_FUNC
 extern void ziti_endpoint_state_change(ziti_context ztx, bool woken, bool unlocked);
 
+/**
+ * @brief Extend Certificate Authenticator Callback
+ *
+ * This callback is invoked on the conclusion of ziti_extend_cert_authenticator(). The result of the function
+ * may be an error condition so it is important to verify the status code in this callback. In the
+ * event the certificate authenticator extension request cannot be completed an error will be returned otherwise
+ * #ZITI_OK is expected.
+ *
+ * @see ziti_extend_cert_authenticator(), ZITI_ERRORS
+ */
+typedef void (*ziti_extend_cert_authenticator_cb)(ziti_context ztx, const char* authenticator_id, const char* cert_pem, int status, void *ctx);
```

Comment:
what purpose `authenticator_id` serves here -- there is no function that takes it?
 is the app supposed to use it?

Reply:
Nothing other than logging info, but it doesn't need to be in the CB if the c-sdk logs it. Will remove.